README.md

```diff
@@ -5,9 +5,12 @@
 This is my educational project within IT KPI Python mentorship program by @webknjaz.
 
 ## Installation
-
-Soon
-
+```
+git clone <url>
+```
+```
+pip install requirements.txt
+```
```

Comment:
These instructions should be more accurate — like, having an actual URL. Also, use syntax highlighting. You may choose `shell` or `console` here, depending on how you shape the content.

Reply:
Rewritten in https://github.com/kpi-web-guild/django-girls-blog-Martolivna/pull/19/commits/c32a591aa8a9c97adb244fe4adf7087e816c8643 .

Reply:
Marking as resolved. I'll add relevant comments to the new additions separately.